controllers/k8ssandra/k8ssandracluster_controller_test.go

```diff
@@ -52,18 +47,28 @@ import (
 const (
 	timeout  = time.Second * 5
 	interval = time.Millisecond * 500
+
+	k8sCtx0 = "cluster-0"
+	k8sCtx1 = "cluster-1"
+	k8sCtx2 = "cluster-2"
 )
 
 var (
-	defaultStorageClass = "default"
-	testEnv             *testutils.MultiClusterTestEnv
-	managementApi       = &fakeManagementApiFactory{}
+	defaultStorageClass  = "default"
+	testEnv              *testutils.MultiClusterTestEnv
+	managementApiFactory = &testutils.FakeManagementApiFactory{}
+
+	systemKeyspaces = []string{"system_auth", "system_distributed", "system_traces"}
```

Comment:
Shouldn't we promote this to a global var in production code? Seems error prone to re-declare this list each time we need it.

Reply:
I see similar slices in:
* controllers/k8ssandra/replication.go:90
* controllers/k8ssandra/replication.go:203

Reply:
Actually, a global var makes me nervous that it would be error prone since the slice isn't immutable. How about instead a util function that always returns a new copy?

Reply:
That's a very interesting remark. 

We considered this in go-cassandra-native-protocol a while ago. This project exports a lot of global vars, and we were concerned by the fact that these variables were mutable. After some research we came to the conclusion that we shouldn't care about this. If a consumer of your package alters any exported var, it's their fault. The runtime packages themselves export a bunch of mutable variables. If you want to break almost every application out there, simply do this:

```go
import "time"
func init() {
	time.UTC = nil
}
```

That said, I don't mind using the function approach either, if you prefer to err on the safe side.

Reply:
I'm good with global var :)